src/olympia/devhub/tests/test_views_submit.py

```diff
@@ -39,21 +39,19 @@ def setUp(self):
 
     def get_img_urls(self):
         return (
-            reverse('devhub.personas.upload_persona', args=['persona_header']),
-            reverse('devhub.personas.upload_persona', args=['persona_footer'])
+            reverse('devhub.personas.upload_persona', args=['persona_header'])
```

Comment:
This isn't a tuple anymore, if you add a `,` at the end it's a tuple again and the tests should work again.

```python
return (
    reverse('devhub.personas.upload_persona', args=['persona_header']),
)
```